xml/System/Span`1.xml

```diff
@@ -28,9 +28,14 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="T">The type of items in the <see cref="System.Span`1" />.</typeparam>
+    <summary>Represents a contiguous region of arbitrary memory that is type- and memory-safe.</summary>
```

Comment:
This makes it sound like the memory being represented is type- and memory-safe, rather than it being the representation that is type- and memory-safe.